plenum/recorder/src/silencer.py

```diff
@@ -0,0 +1,16 @@
+from typing import Tuple, Optional, Union
+
+from stp_zmq.remote import Remote
+
+
+class Silencer:
```

Comment:
Why do we need Silencer? Where is it used?

Reply:
In replay, to skip trying to send any message on the sockets